src/tensor_ops.jl

```diff
@@ -1,3 +1,15 @@
+# Deprecate `*` and `'*` for tensor multiplication
```

Comment:
These are not really deprecations. Deprecation means that it should still give the right answer. Here it just errors.

This is really good to have though so you can just keep it.